cpp/zotaut/zeder_to_zotero_importer.cc

```diff
@@ -170,7 +170,7 @@ void DownloadZederEntries(const Zeder::Flavour flavour, const std::unordered_set
         new Zeder::FullDumpDownloader::Params(endpoint_url, entries_to_download, columns_to_download, filter_regexps));
 
     auto downloader(Zeder::FullDumpDownloader::Factory(Zeder::FullDumpDownloader::Type::FULL_DUMP, std::move(downloader_params)));
-    if (not downloader->download(downloaded_entries))
+    if (not downloader->download(downloaded_entries, true))
```

Comment:
If we pass a bool to a function, our convention is to add a short comment to see its name, e.g. `/*disable_cache_mechanism=*/ true`

Reply:
so one the one hand we make it more verbose by adding the comment one the other hand it is far less verbose by using default parameters.